index.ts

```diff
@@ -0,0 +1,8 @@
+/// <reference path="src/types/global.d.ts" />
+
+export { ARSession } from './src/components/AR/AR-session';
```

Comment:
Please always use `.js` at the end of imports, **even** when importing a `.ts` file. This is really important because if someone consumes the library as a raw ES6 module without bundling, you will get a bunch of missing file

Reply:
@DavidPeicho 
you mean `export { ARSession } from './src/components/AR/AR-session.js'`;

Reply:
Yes, same with all imports